std/stdio.d

```diff
@@ -3201,20 +3199,20 @@ struct LockingTextReader
     private char _front;
     private bool _hasChar;
 
-    this(File f)
+    this(File f) @trusted
     {
```

Comment:
Maybe you could add your reasoning from the [NG thread](http://forum.dlang.org/thread/jlpszfehknqwsjbakwqf@forum.dlang.org#post-8a16f413-be66-cb53-1ef3-460021de9c78:40erdani.com) so that in the future people don't have to wonder about the use of `@trusted` here.